src/style.css

```diff
@@ -1,3 +1,69 @@
+:root {
+  --main-color:  #00ACDC;
+}
+
 body {
   margin: 0;
+  font-family: Roboto, sans-serif;
+}
+
+.header {
+    display: flex;
+    align-items: center;
+    justify-content: space-between;
+    box-sizing: border-box;
```

Comment:
set box-sizing for all elements or remove it

Reply:
Could you clarify please? Don't understand, if it's work, why I should replace it. 
Deleting nit working for this =((

Reply:
Done =)